MODULE.bazel

```diff
@@ -0,0 +1,90 @@
+module(name = "publish-to-bcr")
```

Comment:
nit: if this is a leaf, then we don't need this line, it should not be referenceable by modules that want to depend on it